openmls/src/test_utils/mod.rs

```diff
@@ -7,6 +7,8 @@ use std::{
     io::{BufReader, Write},
 };
 
+pub mod test_framework;
+
 pub fn write(file_name: &str, obj: impl Serialize) {
```

Comment:
A  little unrelated, but can you check that all these functions in here need to be public?

Reply:
Done. All of these functions are crate-public now.